packages/app/src/cli/services/import-extensions.ts

```diff
@@ -15,7 +15,7 @@ interface ImportOptions {
   apiKey?: string
   developerPlatformClient?: DeveloperPlatformClient
   extensionTypes: string[]
-  buildTomlObject: (ext: ExtensionRegistration) => string
+  buildTomlObject: (ext: ExtensionRegistration, appModules: ExtensionRegistration[]) => string
```

Comment:
`appModules` here represents all of the extensions, just like the `ext` right? If so, why are we calling it an app module and the single extension `ext`? In order to be consistent, let's either call the new field something like "allExtensions" or rename the `ext` into an `appModule`. I personally prefer the former